components/header/src/overrides/header.js

```diff
@@ -0,0 +1,39 @@
+/** @jsx jsx */
+
+import { jsx, getLabel, useMediaQuery } from '@westpac/core';
+
+// ==============================
+// Component
+// ==============================
+
+const Header = ({ state: _, ...rest }) => <header {...rest} />;
+
+// ==============================
+// Styles
+// ==============================
+
+const headerStyles = (_, { fixed }) => {
+	const mq = useMediaQuery();
+	return mq({
+		label: getLabel('header'),
+		flex: 'none',
+		display: 'flex',
+		...(fixed && { height: ['calc(3.375rem + 1px)', null, 'calc(4.0625rem + 1px)'] }),
+	})[0];
```

Comment:
One detail. Footer wrapper has white background color, while header wrapper has none - it's INNER wrapper that has background color applied. Consequently, when adding e.g. 

```js
const overrides = {
  Inner: {
    styles: (styles) => ({
      ...styles,
      maxWidth: 1280,
      margin: '0 auto',
    }),
  },
};
``` 

to both header and footer, footer is all white with nicely centered content, while header is transparent with white rectangle awkwardly sitting there.

![image](https://user-images.githubusercontent.com/5426427/128863389-d5e36f1e-aebd-48b6-98fb-6717290c067e.png)